### PR TITLE
fitter 1.7.1 :snowflake:

### PR DESCRIPTION
**Pull request by psteyer**
fitter 1.7.1 :snowflake:

**Destination channel:** Snowflake

### Links

- [PKG-5811]
- dev_url:        https://github.com/cokelaer/fitter/tree/v1.7.1
- conda_forge:    None
- pypi:           https://pypi.org/project/fitter/1.7.1
- pypi inspector: https://inspector.pypi.io/project/fitter/1.7.1

### Explanation of changes:

- new version number

[PKG-5811]: https://anaconda.atlassian.net/browse/PKG-5811?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml
Comment (Arishamays1) [inline recipe/meta.yaml:56]: Adding a description might be a nice touch!

File: recipe/meta.yaml
Comment (cbouss) [inline recipe/meta.yaml:47]:
```suggestion
  summary: A tool to fit data to many distributions and get the best one(s)
  description: |
    The fitter package is a Python library used for fitting probability distributions to data. It provides a 
    straightforward and and intuitive interface to estimate parameters for various types of distributions, both 
    continuous and discrete.
```

File: recipe/meta.yaml
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  doc_url: https://fitter.readthedocs.io/
  dev_url: https://github.com/cokelaer/fitter
  home_url: https://github.com/cokelaer/fitter
  license_family: GPL
```

File: recipe/meta.yaml
Comment (Arishamays1) [inline recipe/meta.yaml]: You might want to add your name here as this is the first PR for this package!

File: abs.yaml
Comment (msentissi) [inline abs.yaml]: I assume these are temporary ? Otherwise LGTM.

**Review by msentissi — APPROVED** [review]

File: recipe/meta.yaml
Comment (msentissi) [inline recipe/meta.yaml:28]: Why is`matplotlib-base` used instead of `matplotlib` ? EDIT: Found this [explanation](https://conda-forge.org/docs/maintainer/knowledge_base/#matplotlib).

**Review by Arishamays1 — APPROVED** [review]
Lgtm

File: abs.yaml
Comment (psteyer) [reply abs.yaml]: Yup! Temporary to make sure it builds with the dependencies I built. They'll get removed before merge.

**Review by cbouss — APPROVED** [review]

File: recipe/meta.yaml
Comment (lorepirri) [reply recipe/meta.yaml:28]: @msentissi it's because `matplotlib-base` is a lighter package for us

**Pull request merged**
